site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -286,27 +286,84 @@ When HTTP POST Bindings are used, this is the default verification key used if a
 +
 This parameter is required when [field]#application.samlv2Configuration.requireSignedRequests# is set to `true`. 
 
-[field]#application.samlv2Configuration.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.6.0#::
-Whether or not the SAML IdP for this Application is enabled or not.
+[field]#application.samlv2Configuration.enabled# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.6.0#::
+Whether or not the SAML IdP for this Application is enabled.
 
 [field]#application.samlv2Configuration.issuer# [type]#[String]# [required]#Required# [since]#Available since 1.6.0#::
-The issuer that identifies the service provider and allows FusionAuth to load the correct Application and SAML configuration. If you don't know the `issuer`, you can often times put in anything here and FusionAuth will display an error message with the `issuer` from the service provider when you test the SAML login.
+An `issuer` identifies the service provider and allows FusionAuth to load the correct Application and SAML configuration. If you don't know the `issuer`, you can put anything in this field and FusionAuth will display an error message with the `issuer` from the service provider when you test the SAML login.
 
 [field]#application.samlv2Configuration.keyId# [type]#[UUID]# [optional]#Optional# [default]#Defaults to a new key# [since]#Available since 1.6.0#::
 The unique Id of the Key used to sign the SAML response. If you do not specify this property, FusionAuth will create a new key and associate it with this Application.
 
+[field]#application.samlv2Configuration.logout.behavior# [type]#[String]# [optional]#Optional# [default]#Defaults to `AllParticipants`# [since]#Available since 1.25.0#::
+The possible values are:
++
+ * `AllParticipants` - each session participant that has enabled single logout will be sent a Logout Request
+ * `OnlyOriginator` - no other session participants will be notified when a logout request is sent for this application
++
+This configuration is functionally equivalent to the Logout Behavior found in the OAuth2 configuration.
+
+[field]#application.samlv2Configuration.logout.defaultVerificationKeyId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.25.0#::
+The unique Id of the Key used to verify the signature if the public key cannot be determined by the `KeyInfo` element when using POST bindings, or the key used to verify the signature when using HTTP Redirect bindings.
++
+This parameter is required when [field]#application.samlv2Configuration.logout.requireSignedRequests# is set to `true`. 
+
+[field]#application.samlv2Configuration.logout.keyId# [type]#[UUID]# [optional]#Optional# [default]#Defaults to the key defined by `application.samlv2Configuration.keyId`# [since]#Available since 1.25.0#::
```

Comment:
We do have a pattern we use sometimes when the default value needs to be described. This is ok as well, but if you want to look at some other examples look for usages of `[default]#defaults to **[see description]**`.